core-plugins/widgets/Joiner-batchjoiner.json

```diff
@@ -0,0 +1,46 @@
+{
+  "metadata": {
+    "spec-version": "1.3"
+  },
+  "inputs": {
+    "multipleInputs": true
```

Comment:
This property means that the plugin allow multiple inputs with different schema. If the config does not have this property, it means that the schema for all inputs have to be the same